src/klein/_plating.py

```diff
@@ -237,21 +237,15 @@ def mymethod(
                     json_data.update(data)
                     for ignored in self._presentationSlots:
                         json_data.pop(ignored, None)
-                    str = "json"
+                    request.setHeader(b"content-type", b"application/json")
```

Comment:
ah I thought you wern't going to make this change?

Reply:
I wasn't but the whole thing just looked broke and I couldn't help it